layouts/shortcodes/render_changelog.html

```diff
@@ -13,11 +13,8 @@ <h3>Changelog</h3>
 
 
 <div id="solodocs-changelogdiv"></div>
-<script>const changelogPath = '{{ (.Get "changelogJsonPath" )}}'</script>
+<script>const changelogPath = '{{.Site.Data.Solo.DocsVersion}}/static/content/{{ .Get "changelogJsonPath" }}'</script>
```

Comment:
Previously, I was setting `changelogJsonPath` as `/static/content/glooe-changelog.docgen` from the `_index.md` in the gloo repo. That is not the correct url, it should be something like `gloo-edge/{VERSION}/static/content/glooe-changelog.docgen`, hence this change.